docs/user/metal/install_ipi.md

```diff
@@ -46,10 +46,19 @@ purposes:
     * `*.apps.<cluster-name>.<base-domain>` - pointing to the Ingress VIP
 
 * **NIC #2 - Provisioning Network**
-  * A private, non-routed network, used for PXE based provisioning.
-  * DHCP is automated for this network.
-  * Addressing for this network is currently hard coded as `172.22.0.0/24`, but
-    will be made configurable in the future.
+  * A private network used for PXE based provisioning.
+  * DHCP is automated for this network by default, to rely on external
+    DHCP, set the platform's `provisioningDHCPExternal` option to `true`
+  * Addressing for this network defaults to `172.22.0.0/24`, but is
+    configurable by setting the `provisioningNetworkCIDR` option.
+  * Two IP's are required to be available for use, one for the bootstrap
+    host, and one as a provisioning VIP in the running cluster. By
+    default, these are the 2nd and 3rd addresses in the
+    `provisioningNetworkCIDR` (e.g. 172.22.0.2, and 172.22.0.3).
+  * Currently, the hardware used for a machine set must be homogenous,
```

Comment:
This last point is a little misleading I think - provisioningNetworkInterface is the nic we bring up on a master for the provisioning services, it's not related to any machineset, and it's not necessarily true that we need the same nic name on all non-master nodes.

I think https://github.com/openshift/installer/issues/2762 also confuses the two cases of nic specification, so it'd be good to be clear here.

Reply:
Don't we though? We don't know where the metal3 pod will run in the cluster, which means we kind of assume that all hardware has access to the provisioning network on the same interface

Maybe I'm a bit confused here

Reply:
Oh: do we guarantee the Metal3 pod only ever runs on the control plane hosts?

Reply:
> do we guarantee the Metal3 pod only ever runs on the control plane hosts?

That is a good point, I'm not sure that we do atm, but probably we should prevent it being potentially rescheduled onto any workers, since we expect to manage all worker lifecycle via the metal3 pod?

Reply:
I think we've not bothered to constrain where it runs because we need to be able to fence control plane nodes, too.